static/styles/portico/landing_page.css

```diff
@@ -1573,6 +1573,20 @@ nav {
     }
 }
 
+.carousel-control.left:hover #prev {
```

Comment:
I meant inside the block.
Something like this,
```css
.tour .carousel-control { /* Line 1533 */
    background: 0;
    border: 0;
    color: hsl(223, 13%, 52%);
    font-size: 2em;
    top: 50%;
    font-weight: 400;
    opacity: 1;
    border-radius: 0;

    &:hover {
        color: hsl(220, 23%, 33%);
    }
+
+    .carousel-label {
+        display: none;
+        font-family: inherit;
+        font-size: 14px;
+    }

```

Reply:
oh, I see...

Reply:
What about the css for hover, that should remain outside right?

Reply:
No see next [comment](https://github.com/zulip/zulip/pull/17844#discussion_r602848985).